test/multiverse/suites/rails/rails_logger_test.rb

```diff
@@ -0,0 +1,43 @@
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+require './app'
```

Comment:
Equivalent to `require_relative 'app'` with fewer keystrokes?

Reply:
Great catch! I prefer require_relative. This was a copy/paste carryover.

Reply:
Updated in: 0f9ebe3eb1545bc8d5279f193e608134ed4e5478